.github/workflows/build.yml

```diff
@@ -0,0 +1,59 @@
+name: "Build / make"
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  make-check:
+    runs-on: ubuntu-latest
+    container: ${{ matrix.container }}
+    strategy:
+      matrix:
+        container: ['ubuntu:latest', 'fedora:latest']
```

Comment:
I just tried whate is faster and it is little bit faster to use our ci image instead of vanilla Fedora.
* fedora-latest 2:17
* quay.io/sssd/ci-client-devel:latest 1:13, one package was updated - it may be faster when there are no package updates

So I guess it really does not matter which one is used as the performance gain is minimal. But it is nice to see there there is some.

However, I just realized that other distributions will be build through intergration tests and copr, so we can just leave this only to detect warnings fast. Therefore we can just run it once on fedora:latest.

Reply:
Just to be clear, you recommend that this workflow run only on `fedora:latest` container, and I should drop the `ubuntu:latest` is that correct?

Reply:
Yes.